### DISCUSSION TITLE
Need help Can someone please let me know if i can change default response using an API than manually

### DISCUSSION THREAD

**Discussion by lav20013** [mockoon/mockoon#876] [Help] [answered]
Noticed I need to change default response by dragging and changing priority.
I have multiple responses and rules for one URL, need to update each one depending on the case.
Can this be update through a PUT API call?

**Comment by 255kb** [comment, marked as answer]
There is currently no way to edit the mocks through an API. Also, endpoints are independent. 
There are some opened issues that you may want to upvote:
Configuration API: https://github.com/mockoon/mockoon/issues/629
CRUD endpoints (on the roadmap): https://github.com/mockoon/mockoon/issues/124

**Reply by lav20013** [reply]
Thank you waiting for update operations to be available
